core/src/main/java/hudson/security/SecurityRealm.java

```diff
@@ -308,6 +308,7 @@ public void doLogout(StaplerRequest req, StaplerResponse rsp) throws IOException
         cookie.setSecure(req.isSecure());
```

Comment:
This already sets the Secure flag when Jenkins is configured to be served via HTTPS.

Reply:
Yes indeed!

:bowing_man: